### ISSUE TITLE
[a11y] Footer - Mobile version - expand button links aria missing

### ISSUE THREAD

**Issue by fffan64** [AxaFrance/design-system#1219]
![Image](https://github.com/user-attachments/assets/bf04f02f-4970-480e-a145-852aeb664169)
